### PR TITLE
Terraform 0.14 upgrade

### PR DESCRIPTION
**Pull request by maximmi**
## what
- Upgrade to support Terraform 0.14 and bring up to current Cloud Posse standard

## why
- Support Terraform 0.14

### CODE REVIEW

**Comment by cocotton** [comment]
Is it possible to bump the `lb-s3-bucket` module version to `0.10.0`? Thanks ^^
https://github.com/cloudposse/terraform-aws-alb/blob/3a3a34f707c5786dd069f6984cb7a513acc7d4ff/main.tf#L41-L43

File: examples/complete/main.tf [hunk 1]
```diff
@@ -3,13 +3,15 @@ provider "aws" {
 }
 
 module "vpc" {
-  source     = "git::https://github.com/cloudposse/terraform-aws-vpc.git?ref=tags/0.17.0"
+  source     = "cloudposse/vpc/aws"
+  version    = "0.17.0"
```
Comment (aknysh) [inline examples/complete/main.tf:7]:
```suggestion
  version    = "0.18.2"
```

File: examples/complete/main.tf [hunk 2]
```diff
@@ -3,13 +3,15 @@ provider "aws" {
...
   cidr_block = var.vpc_cidr_block
   context    = module.this.context
 }
 
 module "subnets" {
-  source               = "git::https://github.com/cloudposse/terraform-aws-dynamic-subnets.git?ref=tags/0.30.0"
+  source               = "cloudposse/dynamic-subnets/aws"
+  version              = "0.30.0"
```
Comment (aknysh) [inline examples/complete/main.tf:14]:
```suggestion
  version              = "0.34.0"
```

**Comment by aknysh** [comment]
> Is it possible to bump the `lb-s3-bucket` module version to `0.10.0`? Thanks ^^
> https://github.com/cloudposse/terraform-aws-alb/blob/3a3a34f707c5786dd069f6984cb7a513acc7d4ff/main.tf#L41-L43

@cocotton  updated

**Review by aknysh — APPROVED** [review]

**Pull request merged**